### ISSUE TITLE
CsSimConnect.csproj - AssemblyOriginatorKeyFile CsSimConnect.pfx is missing - MSB3322	Unable to get MD5 checksum for the key file

### ISSUE THREAD

**Issue by PixelHunter88** [bert-laverman/CsSimConnect#6]
In the CsSimConnect.csproj is a CsSimConnect.pfx file referenced, which is missing.
The compiler refused to build the project, with this error: 
`Error	MSB3322	Unable to get MD5 checksum for the key file Could not find file '..\CsSimConnect-main\CsSimConnect\CsSimConnect.pfx'. `

It was no problem for me to remove the reference in the csproj file.

**Comment by bert-laverman** [comment]
Yes, I think I'm going to remove them all, as there is no real benefit.
